packages/types/lib/persist/api.ts

```diff
@@ -0,0 +1,17 @@
+import type { MergingStrategy } from '../record/api.js';
+
+export interface PostRecordsSuccess {
+    nextMerging: MergingStrategy;
+}
+
+export interface PutRecordsSuccess {
+    nextMerging: MergingStrategy;
+}
+
+export interface DeleteRecordsSuccess {
+    nextMerging: MergingStrategy;
+}
```

Comment:
I could have a single one. wdyt?

Reply:
I'm good like that, stuff can evolve differently